assets/js/theme/common/product-details.js

```diff
@@ -108,15 +110,21 @@ export default class ProductDetails {
             if (type === 'set-rectangle' || type === 'set-radio' || type === 'swatch' || type === 'input-checkbox' || type === 'product-list') {
                 const checked = value.querySelector(':checked');
                 if (checked) {
+                    const isBrowserIE = navigator.userAgent.includes('Trident');
```

Comment:
I suggest to create a separate util file with IE helpers which can be used across app

Reply:
done